src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,75 @@
+// import { render, screen, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import calculate from '../logic/calculate';
+
```

Comment:
- [ ] Please also add test cases for "+/-" and "/" as requested by the previous CR, also, don't forget a test for division of a number by 0 and negative tests.